yarn.config.cjs

```diff
@@ -488,20 +490,35 @@ async function expectWorkspaceLicense(workspace) {
 function expectCorrectWorkspaceExports(workspace) {
```

Comment:
Would there be a way to add the same rules for something like `'exports.*.import.types'`? Some packages have additional subpaths in the exports field, and we may need to enforce that those expose dual CJS/ESM builds as well.

e.g. https://github.com/MetaMask/core/blob/main/packages/notification-services-controller/package.json

Reply:
I made the minimal modifications to the constraints to have a valid main entry point for now.  This would certainly be possible, but I feel like it's out of scope for this PR. What do you think?

Reply:
Agreed! We can take that on in a separate ticket along with the necessary fixes to those exports subfields.

Reply:
Created a ticket here: https://github.com/MetaMask/core/issues/4699